imagebuildah/build.go

```diff
@@ -20,6 +21,7 @@ import (
 	"github.com/containers/storage/pkg/archive"
 	specs "github.com/opencontainers/runtime-spec/specs-go"
 	"github.com/openshift/imagebuilder"
+	dockerParser "github.com/openshift/imagebuilder/dockerfile/parser"
```

Comment:
Can use `"github.com/openshift/imagebuilder/dockerfile/parser"`, and replace *dockerParser.Node with just `*parser.Node`

Reply:
Thanks!